model/map-jpa/src/main/resources/META-INF/groups/jpa-groups-changelog-1.xml

```diff
@@ -36,7 +36,7 @@ limitations under the License.
         </createTable>
         <ext:addGeneratedColumn tableName="kc_group">
             <ext:column name="entityversion" type="INTEGER" jsonColumn="metadata" jsonProperty="entityVersion"/>
-            <ext:column name="realmid" type="VARCHAR(36)" jsonColumn="metadata" jsonProperty="fRealmId"/>
+            <ext:column name="realmid" type="VARCHAR(4000)" jsonColumn="metadata" jsonProperty="fRealmId"/>
             <ext:column name="name" type="VARCHAR(255)" jsonColumn="metadata" jsonProperty="fName"/>
             <ext:column name="parentid" type="VARCHAR(36)" jsonColumn="metadata" jsonProperty="fParentId"/>
```

Comment:
This should be changed as well. Perhaps worth checking all `VARCHAR(36)` fields

Reply:
I thought about that but in my opinion this one should be left as is. Reason being that the parentId is referring to a parent group, and since all groups are managed by the same provider (in this case JPA), we shouldn't have a case where a parent group is stored in a different storage, and therefore it should be able to rely on the particular format adopted by that storage. In this case, it can rely on the parent being a group whose id is a UUID, no?

Reply:
This assumption is not correct - all groups are not generally managed by the same provider. Custom group providers (e.g. LDAP) are allowed, and it is possible that a group from one source would be parent to another.

Reply:
Resized the parent column as well.